src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -243,6 +243,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
 
     backendCall.instantiateCommand match {
       case Success(command) => runWithJes(backendCall, command, jesInputs, jesOutputs, runIdForResumption)
+      case Failure(ex: SocketTimeoutException) => throw ex // probably a GCS transient issue, throwing will cause it to be retried
```

Comment:
Hmm, something seems odd about throwing an Exception from inside a Future {...} block but if this is the idiom then no problem.